cmds/wget/wget.go

```diff
@@ -48,13 +61,30 @@ func usage() {
 }
 
 func main() {
-	flag.Parse()
-	if flag.NArg() != 1 {
+	if flag.Parse(); flag.NArg() != 1 {
 		usage()
 	}
 
-	url := flag.Arg(0)
-	if err := wget(url, os.Stdout); err != nil {
-		log.Fatalf("%v\n", err)
+	argURL := flag.Arg(0)
+	if argURL == "" {
+		log.Fatalln("Empty URL")
+	}
+
+	url, err := url.Parse(argURL)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	fileName := "index.html"
+	if *outPath == "" {
+		if url.Path != "" && url.Path[len(url.Path)-1] != '/' {
```

Comment:
This really ought to be provided by the url package but I can't find it if it is ...